app/controllers/concerns/curation_concerns/selects_collections.rb

```diff
@@ -57,8 +51,8 @@ def collections_search_builder_class
   end
 
   def collections_search_builder(access_level = nil)
-    collections_search_builder_class.new(self).tap do |builder|
-      builder.discovery_perms = access_levels[access_level] if access_level
+    collections_search_builder_class.new(self, nil).tap do |builder|
```

Comment:
It's deprecated, but the good stuff is copied into the controller, so please do it for the improvement of the controller